src/app/tests/suites/certification/PICS.yaml

```diff
@@ -8877,6 +8877,53 @@ PICS:
           "Does the device implement sending the ChangeToModeResponse command?"
       id: LWM.S.C01.Tx
 
+    #  Laundry DRYER Mode Cluster
+
+    - label: "Does the device implement the Dryer cluster as a server?"
+      id: DRYERCTRL.S
+
+    - label: "Does the DUT support testing the failed ChangeToMode command?"
+      id: DRYERCTRL.S.M.CAN_TEST_MODE_FAILURE
+
+    - label:
```

Comment:
I don't think this file is meant to have pixits in it.

Reply:
Removed the Pixits